src/components/downloadPage.jsx

```diff
@@ -27,7 +27,8 @@ export function DownloadPage({
   onChangeFilter,
   changePageRequest,
 }) {
-  if (!loggedIn) {
+  let siteName = profile && profile.user_metadata && profile.user_metadata.siteName ? profile.user_metadata.siteName : null;
```

Comment:
Might be able to consolidate the command a bit. (Just a suggestion, your version likely accounts for more edge cases and such)
```suggestion
  let siteName = profile.user_metadata.siteName ? profile.user_metadata.siteName : null;
```

Reply:
Good point! Thanks!